### DISCUSSION TITLE
Deploy with storage is extremely slow

### DISCUSSION THREAD

**Discussion by renan-garcia** [dokku/dokku#5733] [Q&A]
I have a storage mounted with the following path `/mnt/apollodataproduction/uploads:/app/public/uploads`  But when it is active the deploy is extremely slow, sometimes taking more than half an hour. I own this directory a few hundred gigs of files. At the moment it's impossible to do without them, I tried to ignore them in .slugignore :

```
/mnt/apollodataproduction/uploads
/app/public/uploads
/public/uploads
```

But the slowness to deploy continues. Is there any way fix this?

I think maybe it's something about permissions, something related to `chown`, is there a way to speed up the process of assigning permissions? I didn't create the directory using storage:ensure-directory, would it be something related? if yes how do i fix it?

**Comment by renan-garcia** [comment]
As I suspected the process that hangs is related to permissions.
![image](https://user-images.githubusercontent.com/23019605/224337048-6ff8710d-0bc1-42a4-9175-74c9bdf0a511.png)

**Comment by renan-garcia** [comment]
Output of `ps auxf`:
[message-2.txt](https://github.com/dokku/dokku/files/10947514/message-2.txt)

**Comment by josegonzalez** [comment]
You can disable the chowning of mounted volumes for herokuish-deployed applications by setting the `disable-chown` scheduler-docker-local property as shown here: https://dokku.com/docs/deployment/schedulers/docker-local/?h=disable+chown#disabling-chown-of-persistent-storage
